<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>timeIn</class>
 <widget class="QMainWindow" name="timeIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>622</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>31</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(48, 52, 70);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>611</width>
      <height>591</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>10</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #303446;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>10</number>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>170</y>
       <width>281</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #c6d0f5;
font-size: 30px</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>53</y>
        <width>144</width>
        <height>111</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color: #c6d0f5;
font-size: 30px</string>
       </property>
       <property name="text">
        <string>PLAY TIME</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">color: #c6d0f5;
font-size: 30px</string>
       </property>
       <property name="text">
        <string>  00:00:00</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>380</y>
       <width>231</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="start_butt">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 10% ;
background-color: #232634;
color: #c6d0f5;</string>
        </property>
        <property name="text">
         <string>Start </string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="time_butt_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 10% ;
background-color: #232634;
color: #c6d0f5;</string>
        </property>
        <property name="text">
         <string>Add time</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>570</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>16</width>
       <height>561</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>20</y>
       <width>163</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,5,0,0,0,0,0,0,0,5">
      <property name="spacing">
       <number>16</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_9">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 5%;</string>
        </property>
        <property name="text">
         <string>Habit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_10">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 5%;</string>
        </property>
        <property name="text">
         <string>Habit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_11">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 5%;</string>
        </property>
        <property name="text">
         <string>Habit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_12">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 5%;</string>
        </property>
        <property name="text">
         <string>Habit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_13">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 5%;</string>
        </property>
        <property name="text">
         <string>Habit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 5%;</string>
        </property>
        <property name="text">
         <string>Habit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 5%;</string>
        </property>
        <property name="text">
         <string>Habit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 5%;</string>
        </property>
        <property name="text">
         <string>Habit_4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 5%;</string>
        </property>
        <property name="text">
         <string>Habit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_8">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 5%;</string>
        </property>
        <property name="text">
         <string>Habit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 5%;</string>
        </property>
        <property name="text">
         <string>Habit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_7">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 5%;</string>
        </property>
        <property name="text">
         <string>Habit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>31</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 24%;
font-size: 20px</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>80</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 24%;
font-size: 20px</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>130</y>
       <width>30</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 24%;
font-size: 20px</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>185</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 24%;
font-size: 20px</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>338</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 24%;
font-size: 20px</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>540</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 24%;
font-size: 20px</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>235</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 24%;
font-size: 20px</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>287</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 24%;
font-size: 20px</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>390</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 24%;
font-size: 20px</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>440</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 24%;
font-size: 20px</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>490</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 24%;
font-size: 20px</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 5%;</string>
    </property>
    <property name="text">
     <string>Add game</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
